nexus/src/db/datastore/volume.rs

```diff
@@ -29,9 +29,29 @@ use omicron_common::api::external::ResourceType;
 use serde::Deserialize;
 use serde::Serialize;
 use sled_agent_client::types::VolumeConstructionRequest;
+use steno::ActionError;
 use uuid::Uuid;
 
 impl DataStore {
+    // Create an empty VolumeConstructionRequest and convert it to a string.
+    pub async fn volume_create_empty_data(
+        &self,
+        volume_id: Uuid,
+    ) -> Result<String, ActionError> {
+        // Manufacture an empty volume construction request.
```

Comment:
I think I would put this whole function directly into the caller (the saga action).  It doesn't seem like it belongs in the DataStore.  Returning it as a String seems particularly saga-specific.

Reply:
I agree, it does not really need to be in the datastore, as it is an otherwise empty VolumeConstructionRequest I'm creating using the new UUID.

However, If I create it in `svr_create_temp_volume`, I'll need to bring into that file the knowledge of what a VolumeConstructionRequest is, as that is not something that exists yet at that layer.  I'm making some attempt here to not let the internals of Crucible any further than I need them.

Reply:
Updated after discussion in chat.